appveyor.yml

```diff
@@ -28,11 +28,11 @@ environment:
  scons_publisher: NV Access
  # Comment out any of the feature_* variables to disable the respective build feature.
  # They are checked for existence of content, not specific value.
- feature_buildSymbols: True
- feature_uploadSymbolsToMozilla: True
- feature_buildAppx: True
- feature_crowdinSync: True
- feature_signing: True
+ feature_buildSymbols: configured
+ feature_uploadSymbolsToMozilla: configured
+ #feature_buildAppx: configured
+ feature_crowdinSync: configured
+ feature_signing: configured

```

Comment:
do you mind explaining this change?

Reply:
It's about avoiding semantic overloading and the principle of least astonishment. The variable name (eg: feature_buildSymbols) combined with a boolean value (True) suggests a flag that can be toggled on or off. However, the actual behaviour (checking if the variable is set, regardless of its value) doesn't align with this implication. Setting the value to "configured" means you're using something neutral without a standard negation that's commonly used in programming. This was the smallest and most consistent change I could come up with.

Reply:
my understanding is this is a flag that can be toggled on or off. Is this not correct? I think these features come "pre-configured" in a sense. I'm concerned that there is an implication that you need to perform further configuration when toggling these flags.

Reply:
They can be toggled off, but only by commenting the line out entirely. If you set it to False, the value remains enabled. 

Perhaps there's a better value than "configured"?

Reply:
No, that makes sense now. I assumed "false" worked